test/endpoints/baz/baz_metrics_test.go

```diff
@@ -88,7 +88,7 @@ func TestCallMetrics(t *testing.T) {
 		return
 	}
 
-	cg.MetricsWaitGroup.Wait()
+	<-cg.MetricsWaitGroup.Wait
```

Comment:
Actually, I had that way initially, but decided to expose the Channel similar to how sync.Cond exposes the Locker.  I'll change it back!